### ISSUE TITLE
updating quiet hours does not change "required to stop" time

### ISSUE THREAD

**Issue by ericpaulsen** [coder/coder#14725]
reported by one of our strategic customers:

> If updating quiet hours to later (i.e. from 00:00 UTC to 00:00 EDT) it doesn't change an existing running workspaces “required to stop” by time.

**Comment by stirby** [comment]
Duplicate of https://github.com/coder/coder/issues/12612

**Comment by ethanndickson** [comment]
I don't believe this is a duplicate. To fix this, the server would need to re-calculate the autostop times of all workspaces owned by the user when they updated their quiet hours.
